addons/website/static/src/builder/plugins/options/card_image_option_plugin.js

```diff
@@ -16,22 +17,18 @@ const imageRelatedClasses = [
     "flex-lg-row",
     "flex-lg-row-reverse",
 ];
-const imageRelatedStyles = [
-    "--card-img-aspect-ratio",
-    "--card-img-size-h",
-    "--card-img-ratio-align",
```

Comment:
You probably can not remove `--card-img-ratio-align` from this list as you still want this style to be removed from old snippets when removing the cover image :+1: